src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```diff
@@ -99,6 +99,10 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Condition="'$(OutputType)' == 'WinExe' or '$(OutputType)' == 'Exe'" Include="/STACK:$(IlcDefaultStackSize)" />
       <!-- Do not warn if someone declares UnmanagedCallersOnly with an entrypoint of 'DllGetClassObject' and similar -->
       <LinkerArg Include="/IGNORE:4104" />
+      <!-- Opt into CETCOMPAT by default. -->
+      <LinkerArg Condition="'$(ShadowStacksCompatible)' != 'false'" Include="/CETCOMPAT" />
```

Comment:
We should call this property `CETCompat`. It is what it is called in .vcxproj files.

Reply:
Also, I think we may want make it x64-only, just in case it starts doing something for x86. 
According to docs it is accepted for x86 and currently does nothing.

Reply:
> I think we may want make it x64-only, 

Sounds good to me.

Reply:
(I will start doing something for arm64 eventually.)

Reply:
Also `/CETCOMPAT` seems not accepted on arm64